snapcraft/plugins/python.py

```diff
@@ -45,14 +45,15 @@
       The python version to use. Valid options are: python2 and python3
```

Comment:
ignore this whole commit, its only here to allow me to work on, and test, this change without an ugly rebase when this patch merges